frigate/stats.py

```diff
@@ -82,7 +84,96 @@ def get_temperatures() -> dict[str, float]:
     return temps
 
 
-def stats_snapshot(stats_tracking: StatsTrackingTypes) -> dict[str, Any]:
+def get_processing_stats(config: FrigateConfig, stats: dict[str, str]) -> None:
+    """Get stats for cpu / gpu."""
+
+    async def run_tasks() -> None:
+        await asyncio.wait(
```

Comment:
Is the only reason to introduce asyncio stuff here so we can wait for these to run in parallel?

Reply:
Yes, with the cpu and 2 gpus my production server takes ~ 7-10 seconds and with this it's more like 3-5 seconds. There definitely may be better ways to parallelize it or run the actual checks in the background and cache / average the returned results so when /stats is called it can return without waiting.